articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+---
+title: How to create a site-to-site connection using the UKCloud Azure Stack portal | UKCloud Ltd
```

Comment:
It's a bit long, but the title should probably be:
How to create a site-to-site connection between Azure Stack and Public Azure using the UKCloud Azure Stack portal